spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -66,6 +66,26 @@
     end
   end
 
+  context "with the strict option" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.